dietpi/login

```diff
@@ -196,16 +195,15 @@
 				pgrep 'dietpi-software' &> /dev/null && Other_Session_User_Prompt
 
 				# - Start DietPi-Software
-				/DietPi/dietpi/dietpi-software | tee $FP_TMP_DIETPI_FIRSTRUNSETUP_LOG # Sets G_DIETPI_INSTALL_STAGE=2
-				mv $FP_TMP_DIETPI_FIRSTRUNSETUP_LOG $FP_DIETPI_FIRSTRUNSETUP_LOG
+				/DietPi/dietpi/dietpi-software | tee $FP_DIETPI_FIRSTRUNSETUP_LOG # Sets G_DIETPI_INSTALL_STAGE=2
 
 				# - Update install state, to exit loop, if user skipped immediate reboot
 				export G_DIETPI_INSTALL_STAGE=$(</DietPi/dietpi/.install_stage)
 
 			fi
 
 			# Check for second loop and in case inform user, since this indicates an error
-			if ! (( $G_DIETPI_INSTALL_STAGE == 2 )); then
+			if (( $G_DIETPI_INSTALL_STAGE != 2 )); then
```

Comment:
@Fourdee 
Ah this was wanted. `! (( $G_DIETPI_INSTALL_STAGE == 2 ))` includes a check for valid integer.
- If somehow install state got corrupted empty or what, `! (( $G_DIETPI_INSTALL_STAGE == 2 ))` result is true, so error prompts and such is done.
- `(( $G_DIETPI_INSTALL_STAGE != 2 ))` in case of non-integer would result in false, thus the second loop would start silently.

Reply:
@MichaIng 

Ah apologies, will revert